File: benchmarks/sqrt/sqrt.pl
use strict;
use warnings;

my $sum = 0.0;

for my $n (1 .. 100000) {
    my $x = $n * 12345.6789;
    my $guess = $x;

    for (1 .. 20) {
        $guess = 0.5 * ($guess + $x / $guess);
    }

    $sum += $guess;
}

print "$sum\n";

File: benchmarks/sqrt/sqrt.js
let sum = 0

for (let n = 1; n <= 100000; n++) {
	let x = n * 12345.6789
	let guess = x

	for (let i = 0; i < 20; i++) {
		guess = 0.5 * (guess + x / guess)
	}

	sum += guess
}

console.log(sum)
